src/panels/config/entities/entity-registry-settings.ts

```diff
@@ -340,9 +340,20 @@ export class EntityRegistrySettings extends SubscribeMixin(LitElement) {
               @selected=${this._switchAsChanged}
               @closed=${stopPropagation}
             >
-              <mwc-list-item value="switch" selected>
+              <mwc-list-item
+                value="switch"
+                .selected=${this._deviceClass === "switch"}
```

Comment:
It looks like `"switch": "Switch"` was added to translations but not used?

Should we localize this as `ui.dialogs.entity_registry.editor.device_classes.switch.switch`  instead of calling `${domainToName(this.hass.localize, "switch")` since we are referring to the device class?

Reply:
We are referring here kind of to both with "switch": domain and device_class at the same time 😉 

I had added both "switch" and "outlet" to the translation to have a complete list of device classes for the domain in there in case it gets used somewhere else in the frontend.

But I am fine with either approach here.

Reply:
Since we are talking about device classes, my preference would be that they all use the `ui.dialogs.entity_registry.editor.device_classes`